pytensor/gradient.py

```diff
@@ -1675,6 +1675,7 @@ def verify_grad(
     mode: Optional[Union["Mode", str]] = None,
     cast_to_output_type: bool = False,
     no_debug_ref: bool = True,
+    sum_outputs=False,
```

Comment:
Is this needed? Can't you just sum per output and sidestep the NotImplementedError every time?